css/index.css

```diff
@@ -42,8 +116,8 @@ a{
 
        .links{
         width:500px;
-        display:flex;
-        justify-content: space-evenly;
+        display:grid;
+        gap:50px 100px;
```

Comment:
Since we're moving toward a responsive design, I'd recommend refactoring out the use of pixels. Check out the link in the above comment for the reasoning.